web/src/utils/styling.ts

```diff
@@ -32,3 +32,11 @@ export function useBreakpoint(breakpoint: Breakpoint) {
   };
   return useMediaQuery(queries[breakpoint]);
 }
+
+export function useIsMobile() {
+  return !useBreakpoint('sm');
+}
+
+export function useIsBiggerThanMobile() {
+  return useBreakpoint('sm');
+}
```

Comment:
I don't think this is used. Is it there for future use or should we delete it?

Reply:
Good spotting, hoping to do some slack week work to use it!